### PR TITLE
refactor(object-storage.users.add): improvement + user description fix

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-9093`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-9704
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
1. some improvement
2. fix user description issue

### CODE REVIEW

**Review by anooparveti — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.controller.js [hunk 1]
```diff
@@ -66,13 +66,28 @@ export default class PciUsersAddController {
...
 
+  isUserLinkedMode() {
+    return this.model === this.addExistingUser;
+  }
+
+  isUserCreationMode() {
+    return this.model === this.createNewUser;
+  }
+
+  isUserCanBeCreate() {
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.controller.js:84]:
```suggestion
  canUserBeCreated() {
```
using is + can in the same sentence is not correct

File: packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.controller.js [hunk 2]
```diff
@@ -66,13 +66,28 @@ export default class PciUsersAddController {
...
       .finally(() => {
         this.isLoading = false;
       });
   }
 
+  isUserLinkedMode() {
+    return this.model === this.addExistingUser;
+  }
+
+  isUserCreationMode() {
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.controller.js:80]: Imo this function name is lacking something, I don't understand did you mean 
```suggestion
  isUserCreationModeActive() {
```
?

**Review by marie-j — CHANGES_REQUESTED** [review]
Could you please change you commit message it mentions refactor ,improvement and a fix, which one is it ?

File: packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.controller.js (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.controller.js]: done

**Pull request merged**